aesara/version.py

```diff
@@ -1,27 +1,31 @@
-from aesara._version import get_versions
-
-FALLBACK_VERSION = "1.0.5+unknown"
-
-info = get_versions()
-if info["error"] is not None:
-    info["version"] = FALLBACK_VERSION
+try:
+    from aesara._version import __version__ as version
+except ImportError:
+    raise RuntimeError(
+        "Unable to find the version number which is generated when either building or "
+        "installing from source. Please make sure that this Aesara has been properly "
+        "installed, e.g. with\n\n  pip install -e .\n"
+    )
 
-full_version = info["version"]
-git_revision = info["full-revisionid"]
-del get_versions
+deprecated_names = [
+    "FALLBACK_VERSION",
+    "full_version",
+    "git_revision",
+    "short_version",
+    "release",
+]
 
-short_version = full_version.split("+")[0]
 
+def __getattr__(name):
+    # (Called when the module attribute is not found.)
+    if name in deprecated_names:
+        raise RuntimeError(
```

Comment:
It cannot be a warning unless I reimplement the aforementioned variables so that execution can proceed. If you think that those variables are part of Aesara's public interface and shouldn't be removed, then I'll try to reimplement them, but I wanted to default to skipping work that might be unnecessary.

Reply:
Either one is fine; I just wasn't sure which it was supposed to be, given that both type/class objects were present.

Reply:
Great! Sorry for the confusion; having `DeprecationWarning` there was purely an error on my part.